### PR TITLE
docs: Extend metainfo

### PR DESCRIPTION
**Pull request by bragefuglseth**
This PR aims at improving Tweak's presentation in app stores.

- Format metainfo file
- Move `<releases>` to the bottom since it'll inevitably get very long
- Change name and summary to be compliant with Flathub's guidelines.
  The "for COSMIC" part of the name is unnecessary given that it's
  mentioned in the summary.
- Make the description slightly longer and more different from the
  summary
- Add various URLs (for contributing, donating, etc.)
- Add hardware support info (supported screen sizes, internet access,
  input devices

Cheers!

### CODE REVIEW

**Comment by bragefuglseth** [comment]
I've separated the formatting/reordering and the other changes into two commits, to make it easy to see what has changed.

File: res/metainfo.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>dev.edfloreshz.CosmicTweaks</id>
-  <name>Tweaks for COSMIC</name>
```
Comment (edfloreshz) [inline res/metainfo.xml:4]: I'm not sure about this, I'd like people to know this is exclusively for COSMIC.

File: res/metainfo.xml [hunk 2]
```diff
@@ -24,46 +29,53 @@
...
+      </description>
+    </release>
+    <release version="0.1.0" date="2024-09-30">
+      <description>
+        <p>Initial release! &#x1f389;</p>
+        <ul>
+          <li>Save current color scheme</li>
+          <li>Import existing color schemes</li>
+          <li>Download color schemes from cosmic-themes.org</li>
+          <li>Customize the panel and dock</li>
```
Comment (edfloreshz) [inline res/metainfo.xml:76]: Could you remove L24 and replace it with this? I'd like the app description to be more clear on what the app does.

**Review by edfloreshz — CHANGES_REQUESTED** [review]

File: res/metainfo.xml (hunk 1 above)
Comment (bragefuglseth) [reply res/metainfo.xml:4]: That's still indicated through the summary, but I can change it back if you'd like.

File: res/metainfo.xml (hunk 2 above)
Comment (bragefuglseth) [reply res/metainfo.xml:76]: Sure!

File: res/metainfo.xml (hunk 1 above)
Comment (edfloreshz) [reply res/metainfo.xml:4]: Alright, let's do it, if there's confusion or anything we can always change it back.

File: res/metainfo.xml (hunk 2 above)
Comment (edfloreshz) [reply res/metainfo.xml:76]: Thanks!

File: res/metainfo.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>dev.edfloreshz.CosmicTweaks</id>
-  <name>Tweaks</name>
+  <name>Tweaks for COSMIC™</name>
```
Comment (edfloreshz) [inline res/metainfo.xml]: Oops, I was okay with leaving it as just Tweaks

```suggestion
  <name>Tweaks</name>
```

Comment (edfloreshz) [reply res/metainfo.xml]: Ah, I don't know why this thing is not updating, never mind.

**Review by edfloreshz — APPROVED** [review]
LGTM

**Comment by edfloreshz** [comment]
Thanks!

**Pull request merged**